docs/ui/auth/fragments/react/authenticator.md

```diff
@@ -278,6 +278,8 @@ const App = () => (
 
 <ui-component-props tag="amplify-greetings"></ui-component-props>
 
+### withAuthenticator
```

Comment:
Curious what the reason is for moving this outside of the fragment? Better encapsulation?